picoquic/picosocks.c

```diff
@@ -303,7 +308,7 @@ int picoquic_open_server_sockets(picoquic_server_sockets_t* sockets, int port)
             }
             ret = picoquic_socket_set_pkt_info(sockets->s_socket[i], sock_af[i]);
             if (ret == 0) {
-                ret = picoquic_bind_to_port(sockets->s_socket[i], sock_af[i], port);
+                ret = picoquic_bind_to_port(NULL, sockets->s_socket[i], sock_af[i], port);
             }
```

Comment:
Please don't do that. Different applications already use "bind_to_port". If you really believe that it is necessary to bind sockets to addresses, you should create a different call, "bind_to_port_and_ip". You could then rewrite "bind_to_port" to use the new call with a NULL argument. But see the other comments, I don't think this new function is necessary.

Reply:
Also, if you do that, consider something like:
```
int picoquic_bind_to_address(SOCKET fd, struct sockaddr * local_addr)
```
That will be more generic than passing IP addresses as text strings.